plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -542,37 +537,54 @@ export class PersonState {
                 call: this.event.event, // $identify, $create_alias or $merge_dangerously
                 oldPersonIdentified: String(otherPerson.is_identified),
                 newPersonIdentified: String(mergeInto.is_identified),
-                poEEmbraceJoin: String(this.poEEmbraceJoin),
+                poEEmbraceJoin: String(!!this.personOverrideWriter),
             })
             .inc()
         return result
     }
 }
 
-class PersonOverrideWriter {
-    constructor(private postgres: PostgresRouter) {}
+type MergeOperation = {
+    team_id: number
+    old_person_id: string
+    override_person_id: string
+    oldest_event: DateTime
+}
```

Comment:
Good question — it's written this way to match the Postgres table style to avoid adding a layer of translation (or introducing inconsistent style there.) I'll add a comment to clarify.